website/static/js/projectSettings.js

```diff
@@ -67,19 +101,45 @@ var NodeCategorySettings = oop.extend(
                 .then(function(response) {
                     return response.updated_fields.category;
                 })
-                .done(self.updateSuccess.bind(self))
-                .fail(self.updateError.bind(self));
+                .done(self.updateCategorySuccess.bind(self))
+                .fail(self.updateCategoryError.bind(self));
+        },        
+        updateTitle: function() {
+            var self = this;
+            return $osf.putJSON(self.titleDescriptionEditUrl, {
+                    title: $osf.htmlEscape(self.title()),
+                })
+                .done(self.updateDescription.bind(self))
```

Comment:
@lyndsysimon The success handler for updating just the category used to be named 'updateSuccess' but I changed it to specify/distinguish it from the title/description handlers